UI/data/locale/en-US.ini

```
@@ -302,6 +302,11 @@ ConfirmRemove.Title="Confirm Remove"
 ConfirmRemove.Text="Are you sure you wish to remove '$1'?"
 ConfirmRemove.TextMultiple="Are you sure you wish to remove %1 items?"
 
+# confirm paste filter dialog box
+ConfirmPasteFilters.Title="Confirm Paste Filter"
+ConfirmPasteFilters.Text="Are you sure you wish to paste this filter?"
+ConfirmPasteFilters.TextMultiple="Are you sure you wish to paste filters to %1 sources?"
```

This translation is no longer needed.

```suggestion
ConfirmPasteFilters.TextMultiple="Are you sure you wish to paste filters to %1 sources?"
```

---

Done!